mandatory/3-function-output.js

```diff
@@ -3,15 +3,25 @@ function getRandomNumber() {
   return Math.random() * 10;
 }
 
+//The Math.random() function returns a floating-point, pseudo-random number 
+//that's greater than or equal to 0 and less than 1, with approximately uniform 
+//distribution over that range — which you can then scale to your desired range.
+
+
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
+//In JavaScript, concat() is a string method that is used to concatenate strings together.
+// The concat() method appends one or more string values to the calling
+// string and then returns the concatenated result as a new string.
+
```

Comment:
is there any difference if i do it this way? 
```js
    .....
    return word1 + word2;
```

Reply:
If we're talking about these functions:
function ConcatTest(word1, word2) {
    return word1 + word2;
}

function ConcatTest2(word1, word2) {
    return word1.concat(word2)
}

They are the same, there is no difference.

Reply:
yes, that's right. 
one more possible way is this:
```js
function concatTwoWords(word1, word2) {
  return "".concat(word1, word2);
}
```

but, what you gonna het as result here below, when using `concatTwoWords` declared above?
what is the exact output string?

```js
const word1 = "WORD-1";
const word2 = "WORD-2";

concatTwoWords(word1, word2);
```